### ISSUE TITLE
PublicTransitMapper fails with Matsim 13.0

### ISSUE THREAD

**Issue by ksauzz** [matsim-org/pt2matsim#140]
MATSim has [changed the type of TransitRouteStop.getArrivalOffset, and TransitRouteStop.getDepartureOffset from double to OptionalTime](https://github.com/matsim-org/matsim-libs/commit/612cb2f1c938a07b5e39040210d310f1184ef587#diff-172c6433f7cbcab55ba0a1c89b1a53e5f0fc81027ee93cdded4cad53c9f61cebL36-R38), PublicTransitStop.java needs to follow the change.

Error message

```
Exception in thread "main" java.lang.NoSuchMethodError: 'double org.matsim.pt.transitSchedule.api.TransitRouteStop.getArrivalOffset()'
        at org.matsim.pt2matsim.mapping.linkCandidateCreation.PublicTransitStop.createId(PublicTransitStop.java:51)
        at org.matsim.pt2matsim.mapping.linkCandidateCreation.LinkCandidateCreatorStandard.load(LinkCandidateCreatorStandard.java:115)
        at org.matsim.pt2matsim.mapping.linkCandidateCreation.LinkCandidateCreatorStandard.<init>(LinkCandidateCreatorStandard.java:71)
        at org.matsim.pt2matsim.mapping.PTMapper.run(PTMapper.java:113)
        at org.matsim.pt2matsim.mapping.PTMapper.run(PTMapper.java:100)
        at org.matsim.pt2matsim.mapping.PTMapper.mapScheduleToNetwork(PTMapper.java:81)
        at org.matsim.pt2matsim.run.PublicTransitMapper.run(PublicTransitMapper.java:84)
        at org.matsim.pt2matsim.run.PublicTransitMapper.main(PublicTransitMapper.java:61)
```

**Comment by mrieser** [comment]
OptionalTime was actually introduced in MATSim 12. The latest version of pt2matsim, 21.1, should have this fixed (see #133) and be compatible with MATSim 12, and most likely also with MATSim 13.

**Comment by ksauzz** [comment]
Thank you for the quick response. When I got the error I ran PublicTransitMapper using `gradle run`. So I tried to run PublicTransitMapper using `mvn exec:java`, then it worked. Apparently groovy runtime or something causes my issue. This is not an issue of pt2matsim. I'm going to close this. 
Thanks.
